docs/delivery/T3.md

```
@@ -0,0 +1,16 @@
+# Task T3: SSR Integration
+
+## Objective
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)